lighthouse-core/gather/gatherers/dobetterweb/response-compression.js

```diff
@@ -40,7 +40,7 @@ class ResponseCompression extends Gatherer {
     const unoptimizedResponses = [];
 
     networkRecords.forEach(record => {
-      // Ignore records from OOPIFs
+      // Ignore records from child targets (OOPIFS).
       if (record.sessionId) return;
```

Comment:
requests from the main target still won't have a `sessionId` will they?

Reply:
Correct.